.github/workflows/performance.yml

```diff
@@ -0,0 +1,151 @@
+          sed -i 's/\/tmp\/wasmtime_commit.so/patch/g' /tmp/results_cycles_summarized.csv
+          sed -i 's/\/tmp\/wasmtime_main.so/main/g' /tmp/results_cycles_summarized.csv
+          python3 -c "import pandas as pd; pp = pd.read_csv('/tmp/results_cycles_summarized.csv', usecols=['arch','engine','phase', 'mean'], header=0); pp_sorted = pp.sort_values('phase', ascending=True); print(pp_sorted.to_string(index=False));" > /tmp/results_cycles_summarized_pp_sorted.csv
+          sed -i 's/^/ /' /tmp/results_cycles_summarized_pp_sorted.csv
+          sed -i 's/ \+/|/g' /tmp/results_cycles_summarized_pp_sorted.csv
+          sed -i -z 's/\n/|\n/g' /tmp/results_cycles_summarized_pp_sorted.csv
+          sed -i '2 i\ |-|-|-|-|' /tmp/results_cycles_summarized_pp_sorted.csv
+          sed -i '1 i\ Performance results in clockticks (lower is better):\n' /tmp/results_cycles_summarized_pp_sorted.csv
+
+          #python3 -c "import pandas as pd; pp = pd.read_csv('/tmp/results_cycles_summarized.csv', usecols=['arch','engine','phase', 'mean'], header=0); pp_sorted = pp;  print(pp_sorted.to_string(index=False)); pp_sorted.insert(4, '%Change (1-Head/Patch)',[ 1 -pp_sorted.loc[0][3]/pp_sorted.loc[3][3], 0, 1-pp_sorted.loc[1][3]/pp_sorted.loc[4][3],0, 1-pp_sorted.loc[2][3]/pp_sorted.loc[5][3] ,0], True); print(pp_sorted)"  > /tmp/results_cycles_summarized_pp_sorted2.csv
+          python3 -c "import pandas as pd; pp = pd.read_csv('/tmp/results_cycles_summarized.csv', usecols=['arch','engine','phase', 'mean'], header=0); pp_sorted = pp; pp_sorted.insert(4, '%Change_(1-Head/Patch)',[ 1 -pp_sorted.loc[0][3]/pp_sorted.loc[3][3], 1-pp_sorted.loc[1][3]/pp_sorted.loc[4][3], 1-pp_sorted.loc[2][3]/pp_sorted.loc[5][3],0,0,0], True); pp_sorted.drop('mean', axis=1, inplace=True);  print(pp_sorted.to_string(index=False))"  > /tmp/results_cycles_summarized_pp_sorted2.csv
+          sed -i 's/^/ /' /tmp/results_cycles_summarized_pp_sorted2.csv
+          sed -i 's/ \+/|/g' /tmp/results_cycles_summarized_pp_sorted2.csv
+          sed -i -z 's/\n/|\n/g' /tmp/results_cycles_summarized_pp_sorted2.csv
+          sed -i '2 i\ |-|-|-|-|' /tmp/results_cycles_summarized_pp_sorted2.csv
+          sed -i '/main/d' /tmp/results_cycles_summarized_pp_sorted2.csv
+          sed -i '1 i\ Performance results based on clockticks comparison with main HEAD (higher %change shows improvement):\n' /tmp/results_cycles_summarized_pp_sorted2.csv
+
+     # - id: get-comment-body
+     #   name: Get Result
+     #   run: |
+     #     body="$(cat /tmp/results_cycles_summarized_pp_sorted.csv)"
+     #     body="${body//'%'/'%25'}"
+     #     body="${body//$'\n'/'%0A'}"
+     #     body="${body//$'\r'/'%0D'}"
+     #     echo "::set-output name=body::$body"
+
+      - name: Find PR
+        uses: jwalton/gh-find-current-pr@v1
+        id: findPr
+        with:
+          # Can be "open", "closed", or "all".  Defaults to "open".
+          state: open
```

Comment:
I think we might be able to detect this with various env vars set on github actions for prs?

Reply:
Although if not my main concern about using third-party actions is the security implications. AFAIK we're largely giving all these actions read/write access to the wasmtime repo so a push to this jwalton/gh-find-current-pr repo could end up compromising us. 

One way to mitigate this perhaps would be to run this workflow in a separate repository that is triggered from this repository (or something like that). That way if our other repository gets messed up it's not the end of the world.

Reply:
I think we can get away with not using this findPr or other third-party actions which I'll try to do. That said, when you say "AFAIK we're largely giving all these actions read/write access to the wasmtime repo so a push to this jwalton/gh-find-current-pr repo could end up compromising us." do you mean read/write access to just the downloaded wasmtime repository that is housed on the remote server or are you thinking access to the wasmtime repository as it lives in github?

Reply:
I believe that workflows generally get read/write tokens to the bytecodealliance/wasmtime repository that lives on github, so I'm under the impression that a malicious workflow could compromise big chunks of the source code and repo